hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/upgrade/OMFinalizeUpgradeRequest.java

```diff
@@ -63,11 +64,20 @@ public OMClientResponse validateAndUpdateCache(
 
       String upgradeClientID = request.getUpgradeClientId();
 
-      UpgradeFinalizationStatus status =
+      StatusAndMessages omStatus =
```

Comment:
Line **54** has a debug log line left.

Reply:
Nice catch, it seems I forgot to remove this one. I am pushing the deletion of this line.